extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1021,17 +1091,8 @@ export class AmpA4A extends AMP.BaseElement {
 
     this.isCollapsed_ = false;
 
-    // Allow embed to release its resources.
-    if (this.friendlyIframeEmbed_) {
-      this.friendlyIframeEmbed_.destroy();
-      this.friendlyIframeEmbed_ = null;
-    }
-
     // Remove rendering frame, if it exists.
-    if (this.iframe && this.iframe.parentElement) {
-      this.iframe.parentElement.removeChild(this.iframe);
-      this.iframe = null;
-    }
+    this.destroyFrame();
```

Comment:
We took deliberate steps to avoid keeping a reference to the RefreshManager in AmpA4A, so it may be difficult to cancel/restart refresh on demand.